lib/run_action.js

```diff
@@ -6,7 +6,21 @@ var objectToQueryParamString = require('./object_to_query_param_string');
 // This will become require('xhr') in the browser.
 var request = require('request');
 
+/*
+ * "Full Jitter" algorithm taken from https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/
```

Comment:
Nit: because this is just one line, we can use a `//` comment.

Reply:
👍